### PR TITLE
Typo "STOPAT**=**_time_**,**"→"STOPAT **=** _time_ **,**"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/sql/relational-databases/backup-restore/restore-a-sql-server-database-to-a-point-in-time-full-recovery-model?view=sql-server-ver15

### CODE REVIEW

**Comment by PRMerger9** [comment]
@hyoshioka0128 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/backup-restore/restore-a-sql-server-database-to-a-point-in-time-full-recovery-model.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ ms.author: chadam
   
 3.  Restore the last differential database backup, if any,  without recovering the database (RESTORE DATABASE *database_name* FROM *backup_device* WITH NORECOVERY).  
   
-4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (RESTORE DATABASE *database_name* FROM <backup_device> WITH STOPAT**=**_time_**,** RECOVERY).  
+4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (RESTORE DATABASE *database_name* FROM <backup_device> WITH STOPAT **=** _time_ **,** RECOVERY).
```
Comment (MikeRayMSFT) [inline docs/relational-databases/backup-restore/restore-a-sql-server-database-to-a-point-in-time-full-recovery-model.md:153]: @cawrites

I wonder if we should try a more standard way to identify text that needs ot be replaced. ```suggestion
4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (`RESTORE DATABASE <database_name> FROM <backup_device> WITH STOPAT = <time>, RECOVERY`).  
```

**Comment by hyoshioka0128** [comment]
@MikeRayMSFT Thanks.

**Comment by ktoliver** [comment]
Hi @MikeRayMSFT @cawrites - Could you let us know whether the pull request should be merged? Thanks.

-PR review team

**Review by cawrites — APPROVED** [review]

**Pull request merged**